### ISSUE TITLE
NTR: Mumps virus RT-PCR in blood positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8100]
New term request Mumps virus RT-PCR in blood positive (nadea3_190513002928): Reverse transcriptase polymerase chain reaction; RNA

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000900
- label: Positive mumps virus PCR test in the blood circulation
- parents: [Positive bloodstream nucleic acid pathogen test (HP:0430067)](https://hpo.jax.org/app/browse/term/HP:0430067)
- pmid: 
- def: Detection of nucleic acid of the mumps virus in the blood circulation by a method such as polymerase chain reaction (PCR) or reverse transcriptase (RT)-PCR.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8100
- synonyms:
